src/swap/components/SwapProvider.tsx

```diff
@@ -10,14 +10,12 @@ import { isSwapError } from '../core/isSwapError';
 import { useSwapBalances } from './useSwapBalances';
 import { getSwapQuote } from '../core/getSwapQuote';
 import { formatTokenAmount } from '../../utils/formatTokenAmount';
-import type {
-  SwapError,
-  SwapErrorState,
-  BuildSwapTransaction,
-  SwapContextType,
-} from '../types';
+import type { SwapError, SwapErrorState, SwapContextType } from '../types';
 import type { Token } from '../../token';
-import type { Address } from 'viem';
+import type { Address, TransactionReceipt } from 'viem';
+import { useSendTransaction, useConfig, type BaseError } from 'wagmi';
+import { waitForTransactionReceipt } from '@wagmi/core';
+import { USER_REJECTED_ERROR_CODE } from '../constants';
```

Comment:
This all order of imports feels off.

it should be
```
// we do all the external libraries
import { waitForTransactionReceipt } from '@wagmi/core';
import {
  createContext,
  useContext,
  useCallback,
  useMemo,
  useState,
} from 'react';
import { useSendTransaction, useConfig } from 'wagmi';

// than all the ./ level
import { useSwapBalances } from './useSwapBalances';
import { buildSwapTransaction } from '../core/buildSwapTransaction';

// than all the ../ level
import { isSwapError } from '../core/isSwapError';
import { getSwapQuote } from '../core/getSwapQuote';
import { USER_REJECTED_ERROR_CODE } from '../constants';

// than all the ../../
import { formatTokenAmount } from '../../utils/formatTokenAmount';

// than the types
import type { Address, TransactionReceipt } from 'viem';
import type { BaseError } from 'wagmi';
```

and ofcourse not comments and no spaces.

Reply:
done